src/graphics/graphics-device.js

```diff
@@ -3,6 +3,8 @@ import { EventHandler } from '../core/event-handler.js';
 import { now } from '../core/time.js';
 import { Debug } from '../core/debug.js';
 import { platform } from '../core/platform.js';
+import { WebglVertexBuffer } from './webgl/webgl-vertex-buffer.js';
```

Comment:
So in future this will go into `WebglGraphicsDevice.js`?

Reply:
yes